src/CMakeLists.txt

```diff
@@ -68,11 +68,9 @@ foreach(kind ${kinds})
   target_link_libraries(${lib_name}_c PRIVATE ${PNG_LIBRARIES})
 
   add_library(${lib_name} STATIC $<TARGET_OBJECTS:${lib_name}_f> $<TARGET_OBJECTS:${lib_name}_c>)
-  if(bacio_VERSION GREATER_EQUAL 2.5.0)
-    target_link_libraries(${lib_name} PUBLIC bacio::bacio ${PNG_LIBRARIES} ${JASPER_LIBRARIES})
-  else()
-    target_link_libraries(${lib_name} PUBLIC bacio::bacio_4 ${PNG_LIBRARIES} ${JASPER_LIBRARIES})
-  endif()
+
+  target_link_libraries(${lib_name} PUBLIC bacio::bacio ${PNG_LIBRARIES} ${JASPER_LIBRARIES})
```

Comment:
Why did you change the handling of bacio names?

Reply:
@edwardhartnett It's handled here: https://github.com/NOAA-EMC/NCEPLIBS-g2/blob/b05a08644ed8ce638ad579240c1f61df2625ce23/CMakeLists.txt#L29-L31